cfn-resources/markdown-generator/generate_version_markdown.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+)
+
+const mongoDBPublisherID = "bb989456c78c398a858fef18f2ca1bfc1fbba082"
+const ouputMarkdownFile = "resource-versions.md"
+
+func main() {
+	// can potentially get regions from github publish yaml file
+	var regions = []string{"af-south-1", "ap-east-1", "ap-northeast-1", "ap-northeast-2", "ap-northeast-3", "ap-south-1", "ap-southeast-1", "ap-southeast-2", "ap-southeast-3",
```

Comment:
have you considered `go get github.com/aws/aws-sdk-go@latest`? Maybe not for now if you want to move fast with this PR, but perhaps in the future.

Reply:
not sure I understood this point, currently the `aws-sdk-go` has automatic updates by dependabot. We could investigate removing `aws-sdk-go` usages in favor of `aws-sdk-go-v2` outside the scope of this issue.